Improve Readmes (#1)
cyberbotics/webots-cloud-simulation-demos: 5_IDE/README.md

# 5 IDE
This folder contains:
 - A [webots.yaml](webots.yaml) file that allows to enable the IDE in the web simulation.
 - A [worlds](worlds) folder containing the [thumbnail](worlds/.robotino_demo.jpg) (optional, displayed during the simulation load) and the [world](worlds/robotino_demo.wbt).
 - A [controllers](controllers) folder containing the [controller](controllers/robotino3/robotino3.py).

More information about the online IDE is available [here](https://www.cyberbotics.com/doc/guide/setup-a-webots-project-repository#running-a-simulation)

> You can see this simulation on webots.cloud and modify its controller with the IDE [here](https://webots.cloud/run?version=R2022b&url=https://github.com/cyberbotics/webots-cloud-simulation-demos/blob/main/5_IDE/worlds/robotino_demo.wbt)

Reviewer: ```suggestion
You can run this simulation in webots.cloud and modify its controller with the IDE [here](https://webots.cloud/run?version=R2022b&url=https://github.com/cyberbotics/webots-cloud-simulation-demos/blob/main/5_IDE/worlds/robotino_demo.wbt).
```
